.env.example

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_GEMINI_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)